limited/ravenkeeper

```diff
@@ -16,7 +16,7 @@ __Formalized__
 Starting: 
  • Set Counter to floor $total/5
  • Display `Ravens` (Counter)
-Immediate Night: [Condition: @Self->Counter > `0`] |ravenkeeper.1|
+Immediate Night: [Condition: (@Self->Counter > `0`) and (not @Selection is @Self)] |ravenkeeper.1|
```

Comment:
Does this work? Is this tested? This looks like invalid syntax to me. IIRC there only is "not (A)" and "(A) and (B)", so I think it should be: [Condition: (A) and (not (B))] OR [Condition: A, Condition: not (B)]

Reply:
Appears to be resolved by https://github.com/WerewolvesRevamped/Werewolves-Roles/pull/1542/commits/f3b47254c72c0d768bc6f53cde96596f0f0108a4